src/Dapr.Actors/Runtime/ActorManager.cs

```diff
@@ -209,16 +209,13 @@ async Task<byte[]> RequestFunc(Actor actor, CancellationToken ct)
         internal async Task FireTimerAsync(ActorId actorId, Stream requestBodyStream, CancellationToken cancellationToken = default)
         {
             #pragma warning disable 0618
-            var timerData = await JsonSerializer.DeserializeAsync<TimerInfo>(requestBodyStream);
+            var timerData = await JsonSerializer.DeserializeAsync<TimerInfo>(requestBodyStream, jsonSerializerOptions);
```

Comment:
Can you revert this change and add a comment instead? 

TimerInfo and other types provided by the Actor SDK are written to have no behavior that changes based on the options.